.github/workflows/frontend-coverage.yaml

```
@@ -0,0 +1,42 @@
+  build:
+    runs-on: ubuntu-24.04
+    env:
+      working-directory: ./frontend
+
+    strategy:
+      max-parallel: 4
+      matrix:
+        node-version: ["22"]
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)